loaders/src/glTF/2.0/Extensions/KHR_lights.ts

```diff
@@ -0,0 +1,125 @@
+/// <reference path="../../../../../dist/preview release/babylon.d.ts"/>
+
+module BABYLON.GLTF2.Extensions {
+    interface IGLTFLight {
+        type: "directional" | "point" | "spot";
+        color: [number, number, number];
+        intensity: number;
+        // Runtime values
+        index: number;
+    }
+
+    interface IKHRLights {
+        lights: IGLTFLight[];
+    }
+
+    interface IGLTFLightReference {
+        light: number;
+        // Runtime values
+        babylonLight: Light;
+    }
+
+    export class KHRLights extends GLTFLoaderExtension {
+        public get name(): string {
+            return "KHR_lights";
+        }
+
+        private applyCommonProperties(light: Light, lightInfo: IGLTFLight): void {
+            if (lightInfo.color) {
+                light.diffuse.copyFromFloats(lightInfo.color[0], lightInfo.color[1], lightInfo.color[2]);
+            } else {
+                light.diffuse.copyFromFloats(1, 1, 1);
+            }
+
+            if (lightInfo.intensity !== undefined) {
+                light.intensity = lightInfo.intensity;
+            } else {
+                light.intensity = 1;
+            }
+        }
+
+        protected _loadScene(loader: GLTFLoader, context: string, scene: IGLTFScene): boolean { 
+            return this._loadExtension<IGLTFLightReference>(context, scene, (context, extension, onComplete) => {
+                if (extension.light >= 0 && loader._gltf.extensions) {
+                    const lightInfo = loader._gltf.extensions.KHR_lights.lights[extension.light];
+                    if (lightInfo.type !== 'ambient') {
+                        return;
+                    }
+
+                    // Use monochromatic hemispheric light to simulate ambient light.
+                    const direction = new Vector3(0, 1, 0);
+                    extension.babylonLight = new BABYLON.HemisphericLight("light", direction, loader._babylonScene);
```

Comment:
Why not using scene.ambientColor?

Reply:
I thought about that but, for that to work, don't all materials have to have an ambient colour specifically set? That would seem to complicate having the glTF just load and work seamlessly.

Reply:
This is more about performance then. Having to compute a global light for all meshes will be a lot just for the sake of it
So we should mention that ambient is expected to be white by default on material

Reply:
Weird. I had tried using `scene.ambientColor` before and couldn't get it to work without explicitly setting the ambientColor to white for a material. However, I tried it again and it's working now...

So... okay, that's done :)